searchlib/src/vespa/searchlib/grouping/groupengine.h

```diff
@@ -123,15 +123,15 @@ class GroupEngine : protected Collect
     size_t getIdBase(GroupRef g) const { return _idByteSize*g; }
 
     using IdList = std::unique_ptr<expression::ResultNodeVector>;
-    typedef vespalib::Array<Children *> GroupBacking;
-    typedef std::vector<double> RankV;
-    typedef vespalib::Array<uint8_t> IdBacking;
+    using GroupBacking = std::vector<Children *>;
+    using RankV = std::vector<double>;
+    using IdBacking = std::vector<uint8_t>;
```

Comment:
Should these vectors use `allocator_large` as well, or is it fine to just put things on the heap?

Reply:
Yes, these lists will should not be so large that direct mmapping is required. Then there will be other issues.